backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/repos/ApplicationRepository.kt

```diff
@@ -151,6 +153,7 @@ object ApplicationRepository {
                 false
             } else {
                 applicationVerification.verifiedDate = Instant.now()
+                applicationVerification.automaticSource = ApplicationVerificationExternalSource.VEREIN360
```

Comment:
I think the source should be passed as param here, because this function is used for all verifications, not only for verein360

Reply:
good catch, thanks. Fixed it.